tests/models/mistral/test_modeling_mistral.py

```diff
@@ -462,39 +462,126 @@ def test_flash_attn_2_inference_padding_right(self):
         self.skipTest("Mistral flash attention does not support right padding")
 
 
-@require_torch
+@require_torch_gpu
 class MistralIntegrationTest(unittest.TestCase):
+    def tearDown(self):
+        torch.cuda.empty_cache()
+        gc.collect()
+
     @slow
     def test_model_7b_logits(self):
+        cuda_major_version = torch.cuda.get_device_capability()[0]
```

Comment:
Not very clear to me what this will give.
For example, is this the cuda tookkit version installed on the VM or the cuda version shipped with the installed torch?  (I won't insist to dive into this at this moment)

Reply:
This will give the type of CUDA device we use (T4 or A10), I clarified a bit through concise a comment at init

Reply:
I changed it to `cuda_compute_capability_major_version`